packages/rum-core/src/domain/rumEventsCollection/view/trackInitialViewTimings.spec.ts

```diff
@@ -136,6 +136,16 @@ describe('trackFirstContentfulPaint', () => {
     lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, FAKE_PAINT_ENTRY)
     expect(fcpCallback).not.toHaveBeenCalled()
   })
+
+  it('should not be present if it happens after 10 minutes', () => {
```

Comment:
why?
it could be more interesting to indicate what we are trying to achieve, meaning discarding timings reported after a long time.